modules/ref/pages/client-settings.adoc

```diff
@@ -9,10 +9,11 @@ include::project-docs:partial$attributes.adoc[]
 [abstract]
 {description}
 
-Beyond the connection string, most settings are changed by calls to the libcouchbase API
-(implemented with Couchbase++ from Node.js SDK 4.0) --
-refer to the https://docs.couchbase.com/sdk-api/couchbase-c-client/group\__lcb-cntl.html[API doc on `lcb_cntl()`]
-and the https://docs.couchbase.com/sdk-api/couchbase-c-client/group__lcb-cntl-settings.html[settings list].
+WARNING: the backend implementation of client settings changed substantially in 4.0 and is not currently fully documented. This will be resolved in a future 4.x release.
```

Comment:
```suggestion
WARNING: The backend implementation of client settings changed substantially in 4.0 and is not currently fully documented.
This will be resolved in a future 4.x release.
```